addons/mail/static/src/models/thread.js

```diff
@@ -1929,6 +1927,9 @@ registerModel({
             default: 0,
         }),
         creator: one('User'),
+        currentPartnerFollower: one('Follower', {
```

Comment:
`followerOfCurrentPartner`?

Reply:
maybe not I'm not sure, ok if you don't want to rename

Reply:
Ok for rename, it's a newly introduced field

Reply:
Changes done